### DISCUSSION TITLE
add nomad and firecracker

### DISCUSSION THREAD

**Discussion by jmls** [dyrector-io/dyrectorio#559] [Ideas]
https://firecracker-microvm.github.io/
https://www.nomadproject.io/

Either / both of these would be an awesome feature.

Firecracker especially, as you can then run docker inside a FC VM without having to have elevated privileges on the  host 
Nomad as it's more powerful than docker, but much easier to use than k8s

**Comment by polaroi8d** [comment]
Hi @jmls,

We would be happy to discuss these features further, but unfortunately, we do not have the time to do so at the moment. Our plan is to create an open roadmap for users, and the features that are needed by the majority of users will be implemented by the @dyrector-io/core-engineers team. Our current priority is to release version 0.4.0 with new features and bug fixes, and to provide support for the docker / podman and k8s environments as best as we can.

In the near future, we will investigate the options for integrating with the things you listed above. Thanks 🙏🏻

**Reply by polaroi8d** [reply]
Created 2 issues for the request, #564 and #563 furthermore, added to the [roadmap](https://github.com/orgs/dyrector-io/projects/2/views/1).
